frontend/packages/dev-console/src/components/monitoring/alerts/MonitoringAlerts.tsx

```diff
@@ -0,0 +1,234 @@
+import * as React from 'react';
+import Helmet from 'react-helmet';
+import * as _ from 'lodash';
+// FIXME upgrading redux types is causing many errors at this time
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore
+import { useSelector, useDispatch } from 'react-redux';
+import { match as RMatch } from 'react-router-dom';
+import {
+  expandable,
+  Table,
+  TableHeader,
+  TableBody,
+  sortable,
+  SortByDirection,
+  cellWidth,
+} from '@patternfly/react-table';
+import { FilterToolbar } from '@console/internal/components/filter-toolbar';
+import { getURLSearchParams, Kebab } from '@console/internal/components/utils';
+import {
+  Severity,
+  StateCounts,
+  AlertState,
+  getAlertsAndRules,
+  Rule,
+  Alert,
+} from '@console/internal/components/monitoring';
+import { monitoringSetRules } from '@console/internal/actions/ui';
+import { RootState } from '@console/internal/redux';
+import { alertState, alertDescription } from '@console/internal/reducers/monitoring';
+import { rules } from './rules';
+import './MonitoringAlerts.scss';
+
+type MonitoringAlertsProps = {
+  match: RMatch<{
+    ns?: string;
+  }>;
+};
+
+enum AlertStates {
+  firing = 'Firing',
+  pending = 'Pending',
+  silenced = 'Silenced',
+}
+
+enum Severities {
+  critical = 'Critical',
+  warning = 'Warning',
+}
+
+const monitoringAlertColumn = [
+  {
+    title: 'Name',
+    cellFormatters: [expandable],
+    transforms: [sortable],
+  },
+  { title: 'Severity', transforms: [sortable, cellWidth(20)] },
+  { title: 'Alert State', transforms: [sortable, cellWidth(20)] },
+  '',
+];
+
+const viewAlertRule = () => ({
+  label: 'View Alerting Rule',
+});
```

Comment:
nit: can we have it as below

```suggestion
const viewAlertRule = {
  label: 'View Alerting Rule',
};
```